cmake/LLVM.cmake

```diff
@@ -432,7 +450,7 @@ macro(custom_try_run_exe SOURCE1 SOURCE2 OUTPUT_VAR RES_VAR)
     execute_process(COMMAND "${OUTF}" RESULT_VARIABLE RESV OUTPUT_VARIABLE ${OUTPUT_VAR} ERROR_VARIABLE EV)
     set(${RES_VAR} ${RESV})
     file(REMOVE "${OUTF}")
-    if(${RESV})
+    if(NOT ${RESV} EQUAL 0)
```

Comment:
What is wrong with `if(${RESV})` here?

Reply:
On Windows/MinGW, I got string values in `RESV` (e.g. `Access violation`, not simple exit codes), making this not behave as intended.